.buildkite/pipeline.yml

```diff
@@ -4,97 +4,100 @@ env:
   VAULT_PATH: "kv/ci-shared/observability-ingest/cloud/gcp"
   DOCKER_REGISTRY: "docker.elastic.co"
 steps:
-  - label: "Unit tests - Ubuntu 22.04"
-    key: "unit-tests-2204"
-    command: ".buildkite/scripts/steps/unit-tests.sh"
-    artifact_paths:
-      - "build/TEST-**"
-      - "build/diagnostics/*"
-      - "coverage.out"
-    agents:
-      provider: "gcp"
-      image: "family/core-ubuntu-2204"
-    retry:
-      manual:
-        allowed: true
+  - group: "Unit tests"
+    key: "unit-tests"
+    steps:
+      - label: "Unit tests - Ubuntu 22.04"
+        key: "unit-tests-2204"
+        command: ".buildkite/scripts/steps/unit-tests.sh"
+        artifact_paths:
+          - "build/TEST-**"
+          - "build/diagnostics/*"
+          - "coverage.out"
+        agents:
+          provider: "gcp"
+          image: "family/core-ubuntu-2204"
+        retry:
+          manual:
+            allowed: true
 
-  - label: "Unit tests - Ubuntu 22.04 ARM64"
-    key: "unit-tests-2204-arm64"
-    command: ".buildkite/scripts/steps/unit-tests.sh"
-    artifact_paths:
-      - "build/TEST-**"
-      - "build/diagnostics/*"
-      - "coverage.out"
-    agents:
-      provider: "aws"
-      imagePrefix: "core-ubuntu-2204-aarch64"
-      diskSizeGb: 200
-      instanceType: "m6g.4xlarge"
-    retry:
-      manual:
-        allowed: true
+      - label: "Unit tests - Ubuntu 22.04 ARM64"
+        key: "unit-tests-2204-arm64"
+        command: ".buildkite/scripts/steps/unit-tests.sh"
+        artifact_paths:
+          - "build/TEST-**"
+          - "build/diagnostics/*"
+          - "coverage.out"
+        agents:
+          provider: "aws"
+          imagePrefix: "core-ubuntu-2204-aarch64"
+          diskSizeGb: 200
+          instanceType: "m6g.4xlarge"
```

Comment:
I know it's the same type of machine as it was before the change but isn't that a bit too much to simply run unit tests? ---> https://aws.amazon.com/ec2/instance-types/m6g/

Reply:
Same concern from my side, I would think that unless those tests are very CPU (and memory) intensive a `m6g.xlarge` (4vCPUs, 16GiB RAM) should be enough?

Reply:
You're right. Set to `m6g.xlarge`. Should be enough